.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -7,10 +7,12 @@ body:
   id: read-troubleshooting-guide
   attributes:
     label: Checks
-    description: Please check the boxes below before submitting
+    description: Please check all the boxes below before submitting
     options:
     - label: I've already read https://github.com/actions-runner-controller/actions-runner-controller/blob/master/TROUBLESHOOTING.md and I'm sure my issue is not covered in the troubleshooting guide.
       required: true
+    - label: I'm not using a custom runner image
```

Comment:
I think this needs wording slightly differently, we expect images built on our base to work fine and I can picture some people thinking custom images built from our images (as per the guide) are "customer" runner images which isn't really what we're trying to capture here right?

Reply:
I'm skeptical if everyone can be 100% sure if their "customizations" added on top of the base image won't break anything that ARC expects. Perhaps it's easier for us if we communicate this way. On the other hand, perhaps it's nicer if we could capture folks who make only simple customizations like adding pkgs and commands to the base image.
Do you have any idea about what would be a good sentence to capture only the latter?

Reply:
@toast-gear I missed your edit, sorry! So..

> I'm not using a custom entrypoint in my runner image?

This sounds good!

Can we also add another field to let the user list up all the modifications made to the custom image, which is required when this option is checked?

Reply:
```suggestion
    - label: I'm not using a custom entrypoint in my runner image
```